crowdfunding/users/serializers.py

```diff
@@ -0,0 +1,51 @@
+from rest_framework import serializers
+from .models import CustomUser
+from django.contrib.auth.password_validation import validate_password
+
+class CustomUserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CustomUser
+        fields = ['id','email', 'username', 'password']
+        id = serializers.ReadOnlyField()
+        username = serializers.CharField(max_length=200)
+        email = serializers.CharField(max_length=200)
+        extra_kwargs = {'password': {'write_only': True}}
+        bookmarked_projects = serializers.ReadOnlyField()
```

Comment:
some of these look like they should be outdented to be part of the serializer class and not the meta class